cms/bundles/views.py

```diff
@@ -51,13 +50,9 @@ def dispatch(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> "HttpRe
         if self.page_to_add.in_active_bundle:
             messages.warning(
                 request,
-                _("Page %(title)s is already in a bundle ('%(bundles)s')")
-                % {
-                    "title": self.page_to_add.get_admin_display_title(),  # type: ignore[attr-defined]
-                    "bundles": get_text_list(
-                        list(self.page_to_add.active_bundles.values_list("name", flat=True)), last_word="and"
-                    ),
-                },
+                f"Page {self.page_to_add.get_admin_display_title()} is already in a bundle ('{
```

Comment:
needs `# type: ignore[attr-defined]`

Reply:
Addressed in [20c4f5d](https://github.com/ONSdigital/dis-wagtail/pull/136/commits/20c4f5daa1e25748ca92bf084aa758b6c855f761)